site/.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["next", "prettier"],
   "rules": {
-    "react/no-unescaped-entities": "off"
+    "react/no-unescaped-entities": "off",
+    "@next/next/no-img-element": "off"
```

Comment:
Tại sao 2 rule này phải off ?

Reply:
cái rule đầu tiên em thấy nó có từ ban đầu, còn rule thứ 2 theo em hiểu là từ Nextjs 11 thì cái Eslint nó hỗ trợ quy tắc này "@next/next/no-img-element". Nó muốn mình sử dụng cái next/image, mà thấy anh bảo không nên dùng component Image của vercel nên em tắt cái rule kia đi

Reply:
Theo a mình có thể thử sử dụng `next/image`. Sử dụng tag `image` thuần của HTML thì các ảnh trên website sẽ không được tối ưu, giảm performance website. 
Em có thể sử dụng `default loader` của `next/image`. Note: thêm thư viện `sharp` để ảnh được tối ưu trên môi trường production. Tham khảo: https://nextjs.org/docs/api-reference/next/image#built-in-loaders

Reply:
Sau khi sử dụng `next/image` thay thế tất cả tag `image` => bỏ rule `@next/next/no-img-element`.
Bỏ tiếp rule `react/no-unescaped-entities` và check lại toàn bộ code xem rule đó có ảnh hưởng đến code không ?